spec/models/piece_spec.rb

```diff
@@ -8,14 +8,15 @@
   end
 
   describe "#move_to!" do
+    @user = User.create(
+      email: 'foobar@foobar.com',
+      screen_name: 'foobar',
+      password: 'foobar',
+      password_confirmation: 'foobar'
+    )
+    @game = @user.games.create(white_player_id: @user.id)
+
     before(:all) do
-      @user = User.create(
-        email: 'foobar@foobar.com',
-        screen_name: 'foobar',
-        password: 'foobar',
-        password_confirmation: 'foobar'
-      )
-      @game = @user.games.create(white_player_id: @user.id)
       FROM_ROW = 3
```

Comment:
I think Jacob wanted these row and col constants to be moved to after line 10. `@user=.... @game...` should stay in the `before` block.